android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/components/DateSelectionBox.kt

```diff
@@ -133,6 +136,31 @@ fun showDateRangePicker(
     }
 }
 
+fun setCustomReflection(
+  dpd: MaterialDatePicker.Builder<Pair<Long, Long>>
+): MaterialDatePicker.Builder<Pair<Long, Long>> {
+  try {
+    val datePickerDialogFields: Array<Field> = dpd::class.java.declaredFields
+    for (datePickerDialogField in datePickerDialogFields) {
+      if (datePickerDialogField.name == "mDatePicker") {
+        datePickerDialogField.isAccessible = true
+        val datePicker = datePickerDialogField[dpd] as DatePicker
+        val datePickerFields = datePickerDialogField.type.declaredFields
+        for (datePickerField in datePickerFields) {
+          if ("mDaySpinner" == datePickerField.name) {
+            datePickerField.isAccessible = true
+            val dayPicker = datePickerField[datePicker]
+            (dayPicker as View).visibility = View.GONE
+          }
+        }
+      }
+    }
```

Comment:
Any reason for using reflection in this implementation?

Reply:
Sorry my bad , I used this code for sample used in local branch, And Mistakenly merged in this branch.
I have removed this.